### PR TITLE
Added an option to install dependency packages

### PR DESCRIPTION
**Pull request by narasimhan-v**
Added an option to install dependency packages.
Will work only for Ubuntu, SLES and yum based distros.

Signed-off-by: Narasimhan V <sim@linux.vnet.ibm.com>

### CODE REVIEW

**Comment by narasimhan-v** [comment]
```
# python avocado-setup.py
04:55:34 INFO    : Check for environment
04:55:34 ERROR   : Please install following dependancy packages nanooo emacs
```
```
# python avocado-setup.py --install-deps
04:55:37 INFO    : Check for environment
04:55:37 WARNING : Installing packages nanooo emacs
04:55:38 ERROR   : nanooo not installed
04:55:39 ERROR   : Some packages not installed
```
```
# tail -20  avocado-wrapper.log 
Last metadata expiration check: 0:42:10 ago on Friday 28 June 2019 04:13:28 AM CDT.
No match for argument: nanooo
Error: Unable to find a match: nanooo
2019-06-28 04:55:38 ERROR   : [helper.py:145] nanooo not installed
2019-06-28 04:55:38 DEBUG   : [helper.py:146] Updating Subscription Management repositories.
Unable to read consumer identity
This system is not registered to Red Hat Subscription Management. You can use subscription-manager to register.
Last metadata expiration check: 0:42:10 ago on Friday 28 June 2019 04:13:28 AM CDT.
No match for argument: nanooo
Error: Unable to find a match: nanooo
2019-06-28 04:55:38 DEBUG   : [helper.py:47] Running yum -y install emacs
2019-06-28 04:55:39 DEBUG   : [helper.py:53] Updating Subscription Management repositories.
Unable to read consumer identity
This system is not registered to Red Hat Subscription Management. You can use subscription-manager to register.
Last metadata expiration check: 0:42:11 ago on Friday 28 June 2019 04:13:28 AM CDT.
Package emacs-1:26.1-5.el8.ppc64le is already installed.
Dependencies resolved.
Nothing to do.
Complete!
2019-06-28 04:55:39 ERROR   : [avocado-setup.py:167] Some packages not installed
# 
```

File: avocado-setup.py [hunk 1]
```diff
@@ -159,6 +159,11 @@ def env_check(disable_kvm):
                 "No dependancy check flag is set, proceeding with bootstrap")
             logger.info("Please install following "
                         "dependancy packages %s", " ".join(not_found))
+        elif args.install_deps:
+            logger.warning("Installing packages %s", " ".join(not_found))
+            if helper.install_packages(not_found):
+                logger.error("Some packages not installed")
```
Comment (PraveenPenguin) [inline avocado-setup.py]: can we print which packages not installed , that help user to identify what was wrong

Comment (narasimhan-v) [reply avocado-setup.py]: @PraveenPenguin that is being printed from the `helper.install_packages()`

Comment (PraveenPenguin) [reply avocado-setup.py]: ah ok sure i over looked that code :(

File: avocado-setup.py [hunk 2]
```diff
@@ -159,6 +159,11 @@ def env_check(disable_kvm):
                 "No dependancy check flag is set, proceeding with bootstrap")
             logger.info("Please install following "
                         "dependancy packages %s", " ".join(not_found))
+        elif args.install_deps:
+            logger.warning("Installing packages %s", " ".join(not_found))
```
Comment (sathnaga) [inline avocado-setup.py]: this can be moved to helper function with logger.debug()

Comment (sathnaga) [reply avocado-setup.py]: this is info one time for all packages, make sense..

Comment (narasimhan-v) [reply avocado-setup.py]: @sathnaga done, please check

**Pull request merged**